js/base/Exchange.js

```diff
@@ -819,6 +819,108 @@ module.exports = class Exchange {
         throw new NotSupported (this.id + ' fetchPermissions() not supported yet')
     }
 
+    getInitialPermissions () {
```

Comment:
Initial permissions = exchange.has, which is why I think the `getInitialPermissions` method isn't really necessary.

Reply:
Initial permissions cant be equal to `exchange.has` , the reason is that `.has` includes all possible unified methods (including private methods), while private methods shouldn't be included in `initial permissions` , as initial permissions are the minimal list of allowed endpoints - public endpoints i mean, which can be still used with lowest permssions (which is without api-keys).
also, there is need to define which methods are thought 'to be public methods' in general. that's why I think it's necessary to be there. 
but if you still think we should remove that, please mention/give me a hint how can i manage to separate public (which should be to set to initial `has` value even with unauthorized users) from private endpoints (which should have `undefined` value even if in `has` they have `true`.